client/src/components/tabs.vue

```diff
@@ -61,17 +67,31 @@ export default class Tabs extends Vue {
   max-width: 1140px !important;
   padding: 0 !important;
 }
+
 #new-tab, #existing-tab {
   min-height: 64px;
+  background-color: $BCgovBlue5;
 }
-.upper-left-border {
+
+.upper-border {
   border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
   min-height: 58px;
   max-height: 58px;
+  margin: 0 2.5px;
 }
-.upper-right-border {
+
+::v-deep .v-tab:before {
+  border-top-left-radius: 4px;
   border-top-right-radius: 4px;
-  min-height: 58px;
-  max-height: 58px;
+}
+
+::v-deep .theme--light.v-tabs > .v-tabs-bar{
+  background-color: transparent;
+}
+
+::v-deep .theme--light.v-tabs > .v-tabs-bar .v-tab:not(.v-tab--active) > .v-icon {
```

Comment:
Whoa!
:)

Reply:
I know 😩 
Manipulating Vuetify to meet the requirements can sometimes get a little 😬

Reply:
I noticed in the Vue docs that some global (CSS?) variables can be set to override some default styles. Not sure if that applies here... Don't worry about it; we can look into it later.